data/src/main/kotlin/com/vultisig/wallet/data/repositories/swap/LiFiQuoteSource.kt

```
@@ -0,0 +1,101 @@
+            try {
+                liFiChainApi.getSwapQuote(
+                    fromChain = srcToken.chain.oneInchChainId().toString(),
+                    toChain = dstToken.chain.oneInchChainId().toString(),
+                    fromToken = fromToken,
+                    toToken = toToken,
+                    fromAmount = request.tokenValue.value.toString(),
+                    fromAddress = request.srcAddress,
+                    toAddress = request.dstAddress,
+                    bpsDiscount = request.bpsDiscount,
+                )
+            } catch (e: Exception) {
```

detekt (1.23.8)

[warning] The caught exception is swallowed. The original exception could be lost.

(detekt.exceptions.SwallowedException)